docs/index.html

```diff
@@ -59,40 +59,40 @@ <h1 id="header-1">Features</h1>
                 <p>Now you won't have to anymore. This package allows you to use the latest version of those utility functions in all PHP_CodeSniffer versions from PHPCS 2.6.0 and up.</p>
             </dd>
 
-            <dt>A number of abstract sniff classes which your sniffs can extend.</dt>
+            <dt>Several abstract sniff classes which your sniffs can extend.</dt>
             <dd>
-                <p>These classes take some of the heavy lifting away for a number of frequently occuring sniff types.</p>
+                <p>These classes take some of the heavy lifting away for some frequently occurring sniff types.</p>
```

Comment:
```suggestion
                <p>These classes take most of the heavy lifting away for some frequently occurring sniff types.</p>
```

Two times `some` in the sentence doesn't read right.